CMakeLists.txt

```diff
@@ -438,7 +438,7 @@ if(WITH_CORE)
     set(QT_MIN_VERSION 6.0.0)
     set(QT_VERSION_BASE "Qt6")
   else()
-    set(QT_MIN_VERSION 5.12.0)
```

Comment:
Mmmm.... We had good reasons to limit this to 5.12... I think we'll need to just ifdef in the replacement here